node/nodejs_intro/README.md

```diff
@@ -69,55 +102,78 @@ Let's take a look at the official Chrome V8 website: [https://developers.google.
 > 7 or later, macOS 10.5+, and Linux systems that use IA-32, ARM, or MIPS processors. V8 can run standalone, or can be 
 > embedded into any C++ application. More information can be found on V8's public wiki.
 
-This is basically Google's open source engine that runs and interprets Javascript on all our Google Chrome web browsers.
+This is the specific browser tool that Google uses to run and interpret Javascript on all our Google Chrome web browsers.
 
-The very smart people at Google then thought: What if we just run this standalone without a browser on a computer? That's how Node.js was born.
+But one day, the very smart people at Google thought: **What if we just run this V8 Javascript Engine on it's own without a browser on a computer? This way, we can use it to write server-side code using Javascript!**
 
-### Node.js Environment
+✨ **That's how Node.js was born.** ✨
+<p>&nbsp;</p>
 
-A web server is just a computer serving data on the internet.
+## Node.js Environment and Addons to Javascript
+In short, a web server is just a computer serving data on the internet.
 
-Google decided, with it's V8 Javascript Engine, it's possible to serve webpages, connect to databases, manage files all on the server side.
+![Fullstack](assets/fullstack.png)
+*Figure: Frontend vs. Backend - Author: Seobility - via [link](https://www.seobility.net/en/wiki/Frontend)*
 
-### Node.js: Addons to Javascript
+Google decided, with it's V8 Javascript Engine, that it's possible to serve webpages, connect to databases, and manage files all on the server side.
 
-Stand alone Javascript on the browser does not have a lot of the capabilities we need to handle Server Side programming.
+But the thing is, stand-alone Javascript in the browser does not have a lot of the capabilities we need to do that server-side programming. Browser-based Javascript, for example, wouldn't expect to connect to a database. So, in order to do these new server-side tasks, Node.js adds many new libraries and additional tools to allow us to do the following:
 
-Node.js includes many new libraries and tools to allow us to do the following:
+1. `Input/Output Buffers`: Handle binary data
```

Comment:
Add a familiar touchpoint for students, they have been using node to do code challenges/build simple apps and test them.

Reply:
e.g. "browser Javascript can make a button on a webpage clickable, but it can't talk to a database"?

Reply:
I believe they won't yet know or understand what databases are yet. 

So make a button clickable to trigger an alert or console log?  I know these are very trivial, not real world examples, but for struggling students it should be something they can hold on to.